agents/base/testing.md

```
@@ -0,0 +1,171 @@
+
+Unit Test Runner: Vitest
+Component Testing: @testing-library/react
+E2E Runner: Playwright
+Coverage: @vitest/coverage-v8
+
+---
+
+# Unit Test Rules
+
+- Test files must be colocated with the source file or placed in a __tests__ directory
+- File naming convention: *.test.tsx or *.test.ts
```

markdownlint-cli2 (0.21.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)